runestone/activecode/js/activecode.js

```diff
@@ -294,7 +298,8 @@ ActiveCode.prototype.createControls = function () {
         var butt = document.createElement("button");
         $(butt).text($.i18n("msg_activecode_run_code"));
         $(butt).addClass("btn btn-success run-button");
-        ctrlDiv.appendChild(butt);
+        if (this.code != "\n")
```

Comment:
ako je ideja da se proveri da li je kod prazan, bolje je pitati: `if(this.code.trim() != ''))`